internal/eventlog/eventlog.go

```diff
@@ -34,3 +34,7 @@ func (e *Event) Complete(t time.Time, outcome enums.EventOutcome, msg string, a
 
 	return e
 }
+
+func (e *Event) Succeed(t time.Time, msg string, a ...any) *Event {
+	return e.Complete(t, enums.EventOutcomeSuccess, msg, a...)
+}
```

Comment:
I think this is not used, right? With the wrapper you always call `w.Event.Complete`.